addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -698,26 +698,24 @@ function factory(dependencies) {
             } else {
                 const authorName = author.nameOrDisplayName;
                 if (channel.channel_type === 'channel') {
-                    // hack: notification template does not support OWL components,
-                    // so we simply use their template to make HTML as if it comes
-                    // from component
-                    const channelIcon = this.env.qweb.renderToString('mail.ThreadIcon', {
-                        env: this.env,
-                        thread: channel,
-                    });
+                    const channelIcon = channel.public === 'private' ? '<i class="fa fa-lock"/> '
+                                                                     : '<i class="fa fa-hashtag"/> ';
```

Comment:
Why did you have to change this part?

Reply:
As before this change, "channelIcon" gives a string of HTML code with div tag so as per spec now we need to send this into content part so it takes new line for div tag which doesn't look good.
![Screenshot from 2020-08-14 11-07-39](https://user-images.githubusercontent.com/54100921/90217380-d6105080-de1e-11ea-8ab1-bc0c58392d30.png)

Reply:
Would be best to find a solution that keeps using `ThreadIcon`.
Maybe it should be refactored to return a `span` instead.

Or if we keep a div, there are solution to make it go inline: using flex on a parent (make a div for the whole header), or having the icon div as display inline or inline-block.